AttributeError: 'Config' object has no attribute '_mypy_results_path'

I get this error when using the plugin v0.10.3 together with xdist 3.5.0
addopts = --numprocesses=auto --mypy

.nox/tests-3-12/lib/python3.12/site-packages/_pytest/main.py:306: PluggyTeardownRaisedWarning: A plugin raised an exception during an old-style hookwrapper teardown.
Plugin: terminalreporter, Hook: pytest_sessionfinish
AttributeError: 'Config' object has no attribute '_mypy_results_path'

Hey thanks for the report!
I haven't been able to repro with Python 3.12, pytest-xdist-3.5.0, and pluggy-1.4.0 :thinking: 
``` console
$ venv/bin/pytest --mypy -n auto demo/test_demo.py 
====================== test session starts =======================
platform linux -- Python 3.12.1, pytest-8.0.0, pluggy-1.4.0
Using --randomly-seed=3813770279
rootdir: /home/dtux/Projects/pytest-mypy
configfile: tox.ini
plugins: xdist-3.5.0, mypy-0.10.3, randomly-3.15.0, forked-1.6.0, cov-2.12.1
8 workers [4 items]     
...F                                                       [100%]
============================ FAILURES ============================
__________________________ test session __________________________
[gw1] linux -- Python 3.12.1 /home/dtux/Projects/pytest-mypy/venv/bin/python
mypy exited with status 1.
============================== mypy ==============================
demo/uncollected.py:1: error: Incompatible types in assignment (expression has type "int", variable has type "str")  [assignment]
Found 1 error in 1 file (checked 1 source file)
==================== short test summary info =====================
FAILED demo/test_demo.py::mypy1.8.0-status
================== 1 failed, 3 passed in 1.39s ===================
```
What version of Pytest are you using?

--- a/src/pytest_mypy.py
+++ b/src/pytest_mypy.py
@@ -327,17 +327,18 @@ class MypyWarning(pytest.PytestWarning):
 
 def pytest_terminal_summary(terminalreporter, config):
     """Report stderr and unrecognized lines from stdout."""
-    try:
-        with open(config._mypy_results_path, mode="r") as results_f:
-            results = MypyResults.load(results_f)
-    except FileNotFoundError:
-        # No MypyItems executed.
-        return
-    if results.unmatched_stdout or results.stderr:
-        terminalreporter.section("mypy")
-        if results.unmatched_stdout:
-            color = {"red": True} if results.status else {"green": True}
-            terminalreporter.write_line(results.unmatched_stdout, **color)
-        if results.stderr:
-            terminalreporter.write_line(results.stderr, yellow=True)
-    os.remove(config._mypy_results_path)
+    if _is_master(config):
+        try:
+            with open(config._mypy_results_path, mode="r") as results_f:
+                results = MypyResults.load(results_f)
+        except FileNotFoundError:
+            # No MypyItems executed.
+            return
+        if results.unmatched_stdout or results.stderr:
+            terminalreporter.section("mypy")
+            if results.unmatched_stdout:
+                color = {"red": True} if results.status else {"green": True}
+                terminalreporter.write_line(results.unmatched_stdout, **color)
+            if results.stderr:
+                terminalreporter.write_line(results.stderr, yellow=True)
+        os.remove(config._mypy_results_path)

--- a/tests/test_pytest_mypy.py
+++ b/tests/test_pytest_mypy.py
@@ -89,6 +89,7 @@ def test_mypy_error(testdir, xdist_args):
     )
     result = testdir.runpytest_subprocess(*xdist_args)
     result.assert_outcomes()
+    assert "_mypy_results_path" not in result.stderr.str()
     result = testdir.runpytest_subprocess("--mypy", *xdist_args)
     mypy_file_checks = 1
     mypy_status_check = 1
@@ -96,6 +97,7 @@ def test_mypy_error(testdir, xdist_args):
     result.assert_outcomes(failed=mypy_checks)
     result.stdout.fnmatch_lines(["2: error: Incompatible return value*"])
     assert result.ret != 0
+    assert "_mypy_results_path" not in result.stderr.str()
 
 
 def test_mypy_annotation_unchecked(testdir, xdist_args):

